hack/install.sh

```diff
@@ -263,8 +285,8 @@ do_install() {
 			exit 0
 			;;
 
-		fedora|centos)
-			$sh_c "cat >/etc/yum.repos.d/docker-${repo}.repo" <<-EOF
```

Comment:
@jfrazelle, @Djelibeybi was this line supposed to go away? didn't we add it a few days ago?